webapp/src/pages/FundingAddCard.js

```diff
@@ -28,55 +27,41 @@ export default function FundingAddCard() {
         .createCardStripe({ token: stripeToken })
         .tap(handleUpdateCurrentMember)
         .then((r) => {
-          setSubmitSuccessful({
-            instrumentId: r.data.id,
-            instrumentType: r.data.paymentMethodType,
-          });
+          if (returnTo) {
+            const checkoutURL = `${returnTo}?instrumentId=${r.data.id}&instrumentType=${r.data.paymentMethodType}`;
```

Comment:
Never do this sort of 'this component only works in the context of a particular use case' solution. It is never the right solution to a problem.

Reply:
The point of `returnTo` is to set up this abstraction over specific use case.

Reply:
Oh sorry this is just a misnaming of the variable, the url is ok. Anyway you'll see an improvement in an upcoming commit.